### PR TITLE
feat: 1.29.2 changelog

### PR DESCRIPTION
**Pull request by vapurrmaid**

### CODE REVIEW

**Review by khorne3 — APPROVED** [review]

File: changelog/1.29.2.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  chart.
+
+### Bug fixes 🐛
+
+- infra: fixed issue where filesystem corruption would stall builds on EC2
+  workspaces.
+- helm: updated ingress template to use correct value names.
+- web: fixed issue where dev URLs did not refresh when updated by other clients
+  (e.g., the CLI).
+
```
Comment (johnstcn) [inline changelog/1.29.2.md:23]:
```suggestion
- web: fixed issue where JetBrains 2022.1 IDE versions would not open in Projector.

```

Comment (code-asher) [reply changelog/1.29.2.md:24]: Thanks for taking care of this!

Comment (johnstcn) [inline changelog/1.29.2.md:24]:
```suggestion
- web: fixed an issue where devURLs created using the `configure` script would not show in the UI
```

**Review by khorne3 — APPROVED** [review]

File: changelog/1.29.2.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- infra: added ability to specify ingress className in the Helm chart.
+- infra: added ability to customize liveness and readiness probes in the Helm
+  chart.
+
+### Bug fixes 🐛
+
+- infra: fixed issue where filesystem corruption would stall builds on EC2
+  workspaces.
+- helm: updated ingress template to use correct value names.
+- web: fixed issue where dev URLs did not refresh when updated by other clients
```
Comment (johnstcn) [inline changelog/1.29.2.md:21]:
```suggestion
```

**Review by johnstcn — APPROVED** [review]

**Review by vapurrmaid — COMMENTED** [review]
Thanks for updating @johnstcn

File: changelog/1.29.2.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+- infra: added ability to specify ingress className in the Helm chart.
+- infra: added ability to customize liveness and readiness probes in the Helm
+  chart.
+
+### Bug fixes 🐛
+
+- infra: fixed issue where filesystem corruption would stall builds on EC2
+  workspaces.
+- helm: updated ingress template to use correct value names.
+  (e.g., the CLI).
```
Comment (johnstcn) [inline changelog/1.29.2.md:21]:
```suggestion
```

**Pull request merged**